hackstar.py

```diff
@@ -50,17 +50,43 @@
     )""",
 ]
 
+__SQL_MIGRATIONS = [
+    # Version 0 -> 1: Remove cover column from song table
```

Comment:
We delete the cover URLs. But we never download them. We need to download the URLs listed in the database before dropping the database column as part of the migration, don't we?